Add files via upload

YuntaoShen opened the pull request:
Based on the Generalised Linear Model (GLM) with different distributions, we apply GLM to the dataset. Compared with result of model, we select variables with p-value smaller than 0.05.

YuntaoShen committed:
Add files via upload

Based on the Generalised Linear Model (GLM) with different distributions, we apply GLM to the dataset. Compared with result of model, we select variables with p-value smaller than 0.05.

Jiazhan-Lou committed:
Delete Rplots_Figure1.png

Wrong Branch allocated.

Jiazhan-Lou committed:
Delete Rplots_Figure2.png

Wrong Branch allocated.

dugarig commented:
Thank you for providing a thorough description of the models and their comparisons. Given that Poisson and negative binomial regression models are based on distinct likelihood functions, comparing their AIC values may not be totally applicable.

YuntaoShen committed:
Update project2 GLM.qmd

I delete the GLM with Poisson distribution since we can’t compare the AIC value between two different models. So i delete it and because our response variable is counting data, we retain GLM with negative binomial distribution.

YuntaoShen committed:
Add files via upload

Combined with result of GLM with Negative Distribution, i add analysis based on the p-value of different variables.
